All_years/04-Social_Care/04-SDS_data.R

```diff
@@ -76,7 +76,7 @@ sds_full_clean <- sds_full_data %>%
   )) %>%
   # Fix sds_end_date is earlier than sds_start_date by setting end_date to be the end of fyear
   mutate(sds_end_date = if_else(
-    sds_start_date >= sds_end_date,
+    sds_start_date > sds_end_date,
```

Comment:
I remember I had a conversation with James, and we decide use > rather than >= in fix_sc_end_date.R. So, just to align the logic here, for consistence. This fix may affect around 4888 out of 58000 records.

Reply:
Good to know, that's a definite difference and should be `sds_start_date > sds_end_date` as you say.